config/manifests/kustomization.yaml

```diff
@@ -1,4 +1,5 @@
 resources:
+- bases/update-service-operator.clusterserviceversion.yaml
```

Comment:
Although the files in the `bases` folder is taken by default at the moment, it is safer to be included explicitly in `resources` as the sample of `operator-sdk` shows.

https://github.com/operator-framework/operator-sdk/blob/e95abdbd5ccb7ca0fd586e0c6f578e491b0a025b/testdata/helm/memcached-operator/config/manifests/kustomization.yaml#L4

I also tried `operator-sdk init` and it has this line too.